bin/resources/GameIndex.yaml

```diff
@@ -720,7 +721,6 @@ PCPX-96322:
     eeClampMode: 2 # Otherwise freezes in various spots, check full intro.
     vuClampMode: 1 # Otherwise camera does not focus correctly on main character.
   gsHWFixes:
-    mipmap: 1
```

Comment:
I thought all of this was cleaned up previously.

Reply:
Nope :P